### PR TITLE
[StaticWebAppsV0] Stable Staging Environment inputs for Static Web Apps

### PR DESCRIPTION
**Pull request by mishapos**
**Task name**: StaticWebAppsV0

**Description**: Added task inputs for managing stable staging environments. 

**Documentation changes required:** Y

**Added unit tests:** (Y/N) N

**Attached related issue:** (Y/N) N

**Checklist**:
- [x] Task version was bumped - please check [instruction](https://github.com/microsoft/azure-pipelines-tasks/tree/master/docs/taskversionbumping.md) how to do it
- [x] Checked that applied changes work as expected

### CODE REVIEW

File: Tasks/AzureStaticWebAppV0/task.json [hunk 1]
```diff
@@ -120,6 +120,22 @@
...
       "required": false,
       "helpMarkDown": "Api token for deployment. Not required if passed as an environment variable"
+    },
+    {
+      "name": "deployment_environment",
+      "type": "string",
+      "label": "Deployment Environment",
+      "defaultValue": "",
+      "required": false,
+      "helpMarkDown": "Azure Static Web Apps environment for this deployment"
```
[the remark this answers is not in the document]
Comment (mishapos) [reply Tasks/AzureStaticWebAppV0/task.json]: Thanks, Anthony! I've changed the descriptions to have a little more info. Let me know what you think! :)

**Comment by Reshmi-Sriram** [comment]
@anatolybolshakov @denis-tikhomirov FYI, kindly help us merge this PR as well as #15996. TIA! :)
cc// @vijayma

**Comment by anatolybolshakov** [comment]
Hi @Reshmi-Sriram sorry, we have access issues at the moment - please reach out @mmrazik to approve/merge

**Comment by Reshmi-Sriram** [comment]
@mmrazik FYI. Thank you!

**Review by mkarmark — APPROVED** [review]

**Comment by mishapos** [comment]
Hi @mmrazik one of the CI pipelines seems to have failed because it's using a deprecated version of the windows host for DevOps pipelines. 

This is the error:
```diff
- ##[error]This is a scheduled windows-2016 brownout. The windows-2016 environment is deprecated and will be removed on April 1st, 2022. For more details, see https://github.com/actions/virtual-environments/issues/5238
- ##[error]The remote provider was unable to process the request.
```
Would you know anyone who can update the DevOps host to `windows-2022` so that the CI can run? Thanks! 😄

**Comment by mmrazik** [comment]
> Hi @mmrazik one of the CI pipelines seems to have failed because it's using a deprecated version of the windows host for DevOps pipelines.
> 
> This is the error:
> 
> ```diff
> - ##[error]This is a scheduled windows-2016 brownout. The windows-2016 environment is deprecated and will be removed on April 1st, 2022. For more details, see https://github.com/actions/virtual-environments/issues/5238
> - ##[error]The remote provider was unable to process the request.
> ```
> Would you know anyone who can update the DevOps host to `windows-2022` so that the CI can run? Thanks! 😄

I believe this was just a brownout yesterday. Let me retry first.

**Review by brhopcra — APPROVED** [review]

**Comment by brhopcra** [comment]
Hi @mmrazik the rerun unfortunately seems to have been unsuccessful, do you know what next steps we could take to get this resolved?

**Comment by mmrazik** [comment]
Working on it. Please bear with us.

**Pull request merged**